dashboard/test/ui/features/student_learning/weblab2/weblab2_preview.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Web Lab 2 Preview
+# Safari 16 throws an error due to a regular expression. These types of regular expressions
+# are supported by our minimum Safari version, 16.6, but are not supported by the version Saucelabs uses.
+# Once we upgrade to 17 we can likely remove no_safari.
+@no_safari
+@no_mobile
+
+Scenario: Web Lab 2 Preview loads
```

Comment:
Should we add an Eyes test too?

Reply:
I was going to hold off on an eyes test until the UI was more finalized! Otherwise we are going to be dealing with constant eyes diffs

Reply:
Ah good thinking!